### ISSUE TITLE
Error: not all arguments converted during string formatting

### ISSUE THREAD

**Issue by rguedes** [PokemonGoF/PokemonGo-Bot#2568]
```

2016-08-04 10:14:58,914 [FollowSpiral] [INFO] [position_update] Walking from (xxxx, yyyy, 0) to (xxxx, yyyy, 0) (69.7497328955 m)
2016-08-04 10:15:26,920 [FollowSpiral] [INFO] [position_update] Walking from (xxxx, yyyy, 0) to (xxxx, yyyy, 0) (69.7497328955 m)
2016-08-04 10:15:58,439 [FollowSpiral] [INFO] [position_update] Walking from (xxxx, yyyy, 0) to (xxxx, yyyy, 0) (175.573326448 m)
Traceback (most recent call last):
  File "/usr/lib/python2.7/logging/__init__.py", line 861, in emit
    msg = self.format(record)
  File "/usr/lib/python2.7/logging/__init__.py", line 734, in format
    return fmt.format(record)
  File "/usr/lib/python2.7/logging/__init__.py", line 465, in format
    record.message = record.getMessage()
  File "/usr/lib/python2.7/logging/__init__.py", line 329, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Logged from file __init__.py, line 547
2016-08-04 10:16:44,993 [PokemonGoBot] [INFO] [login_started] Login procedure started.
2016-08-04 10:16:46,317 [PokemonGoBot] [INFO] [login_successful] Login successful.

```

### Other Information
OS: ubuntu 16.04
Git Commit: 046658444b78447b349ff137629f5da9b8a93922
Python Version: Python 2.7.11+

**Comment by iruy** [comment]
There is no need to fix anything atm, read this https://github.com/PokemonGoF/PokemonGo-Bot/issues/2540

**Comment by JaapMoolenaar** [comment]
> There is no need to fix anything atm, read this #2540

I don't agree, yes we have to wait until the proto guys fix it. But that doesn't mean an issue isn't an issue anymore. If and when they fix it, issues should be resolved. 

Or should we all just wait until it's fixed and then bombard with issues?

**Comment by iruy** [comment]
I mean that any efforts you now make to fix an issue, in the future could result in another fix and so, a waste of time. Devs are working like crazye right now, maybe waiting few hours will be good
